bouncer/util.py

```diff
@@ -61,6 +76,9 @@ def parse_document(document):
         except KeyError:
             pass
 
+    if quote == "":
+        quote = "Annotation for {}".format(document_uri)
```

Comment:
We need to test the changes to this function as well, especially the error paths making sure that it behaves as intended when data is missing in the search index (plus the happy path as well).